db/migrate/20170725111950_change_enterprise_contact.rb

```diff
@@ -0,0 +1,18 @@
+class ChangeEnterpriseContact < ActiveRecord::Migration
+  def up
+    remove_columns :enterprises, :email, :contact
```

Comment:
I'll take another look at it, I noticed late last night that the contact (name) field is being used in different ways in different parts of the codebase. Sometimes it's used for email greetings like "Hello {user}", but there's one part where it looks like the contact name gets used as the name for invoices, which seems a bit strange. 

If that's needed as well, maybe there should be a separate "Billing Name" or something that's more clear for that usage.

It's a shame users don't have required name fields, otherwise we could just use those directly in some of these cases.